research-hub-web/src/styles.scss

```diff
@@ -199,15 +183,29 @@ mat-expansion-panel {
   height: 56px;
 }
 
-.search-results-text {
-  color: $dark-blue !important;
-  font-weight: 500;
-  font-style: italic;
-}
 .chip-avatar {
   margin-right: 16px;
   border-radius: 50%;
 }
+.cdk-virtual-scroll-content-wrapper {
```

Comment:
I think this was only for virtual scrolling on the search results page? Can we remove this?

Reply:
Removed